test/e2e/framework/util.go

```diff
@@ -5059,7 +5059,7 @@ func DumpDebugInfo(c clientset.Interface, ns string) {
 }
 
 func IsRetryableAPIError(err error) bool {
-	return apierrs.IsTimeout(err) || apierrs.IsServerTimeout(err) || apierrs.IsTooManyRequests(err) || apierrs.IsInternalError(err)
```

Comment:
I suspect this change is causing autoscaling tests on GKE to fail: https://k8s-testgrid.appspot.com/sig-autoscaling#gci-gke-autoscaling

Can we add it back safely?

Reply:
InternalError is not a retriable error in a strict sense. It depends on our semantic of it, but I don't see where it says we should retry it. Ref: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/api/errors/errors.go#L291

Also, it doesn't set `RetryAfterSeconds` unlike other retriable ones. But I'm not 100% sure. @wojtek-t - WDYT about it?

Reply:
Autoscaling tests tend to trigger multiple master restarts in almost all scenarios, so I suspect this is why we may need to wait it out

Reply:
That sounds reasonable (if that is indeed what is happening). Maybe then we can add it back - though I still wonder why are we not setting retryAfterSeconds for them?

Reply:
As Shyam wrote above - InternalError is not retryable error per se. It should generally be retried on the client side.
That said, since this is testing library, I'm fine with readding it back with a good comment.

But I would really like to understand what are the reason of those failures and why we do see them.